playground/frontend/lib/pages/standalone_playground/screen.dart

```diff
@@ -37,6 +37,7 @@ import 'widgets/playground_page_footer.dart';
 import 'widgets/playground_page_providers.dart';
 
 class StandalonePlaygroundScreen extends StatelessWidget {
+  static const kAppBarButtonsWidth = 1105;
```

Comment:
Explain how this was obtained. Add a TODO and link to an issue to do some automatic calculation.

Reply:
done